proto/tendermint/types/block.proto

```diff
@@ -5,11 +5,9 @@ option go_package = "github.com/tendermint/tendermint/proto/tendermint/types";
 
 import "gogoproto/gogo.proto";
 import "tendermint/types/types.proto";
-import "tendermint/types/evidence.proto";
 
 message Block {
-  Header                        header      = 1 [(gogoproto.nullable) = false];
-  Data                          data        = 2 [(gogoproto.nullable) = false];
-  tendermint.types.EvidenceList evidence    = 3 [(gogoproto.nullable) = false];
-  Commit                        last_commit = 4;
+  Header header      = 1 [(gogoproto.nullable) = false];
+  Data   data        = 2 [(gogoproto.nullable) = false];
+  Commit last_commit = 4;
```

Comment:
shouldn't this be ` = 3` ?

Reply:
hmmm, good eye. I think it could go either way, provided its the same as what's in core.
https://github.com/celestiaorg/celestia-core/blob/ff54db5d24f005696c17051d8e737c4285eba24c/proto/tendermint/types/block.proto#L11-L12

We should at least put a comment there saying that its reserved or something. John knows more about protobuf, wdyt @adlerjohn Should we reserve it or fix it?

Reply:
[If you want to fix it then it should be fixed in both. ](https://github.com/celestiaorg/celestia-core/blame/812dddc5ce5aa9bc5531f5630f3e30a8f44e6d34/proto/types/block.proto#L13)

Reply:
@adlerjohn does this mean that its okay to leave it, or should we fix it?

Reply:
We could leave it as-is. One of the points of protobuf is to allow things like this.